lbrynet/reflector/client/client.py

```diff
@@ -4,16 +4,40 @@
 Client Handshake (sent once per connection, at the start of the connection):
 
 {
-    'version': 0,
+    'version': 1,
 }
 
 
 Server Handshake (sent once per connection, after receiving the client handshake):
 
 {
-    'version': 0,
+    'version': 1,
 }
 
+Client Descriptor Request:
+
+{
+    'sd_blob_hash': "<sd_blob_hash>",
+    'sd_blob_size': <sd_blob_size>
+}
+
+Server Descriptor Response (sent in response to Client Descriptor Request):
+
+{
+    'send_sd_blob': True|False
+    'needed_blobs': []
+}
+
+If response is 'YES', client may send a Client Stream Descriptor or a Client Info Request.
```

Comment:
-I'll rewrite the documentation - it isn't very clear.

-I see what you mean about not needing to make a {`send_blob`: x...} request before sending a blob out of the list populated by the initial `missing_blobs` response. I'd like to hold off on that for another pr.

-The requests and responses are json, the file transfer is raw.